ospf6d/ospf6_route.c

```diff
@@ -1831,6 +1834,31 @@ void ospf6_brouter_show(struct vty *vty, struct ospf6_route *route)
 		OSPF6_PATH_TYPE_NAME(route->path.type), area);
 }
 
+DEFPY (debug_ospf6_route_all,
```

Comment:
why not just add the 'all' keyword to the existing debug route command? that would eliminate some of the other changes.